pkg/scheduler/lifted/plugins/leafnodedistribution/leafnode_distribution_policy.go

```diff
@@ -0,0 +1,316 @@
+package leafnodedistribution
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/scheduler/framework"
+
+	kosmosv1alpha1 "github.com/kosmos.io/kosmos/pkg/apis/kosmos/v1alpha1"
+	"github.com/kosmos.io/kosmos/pkg/generated/clientset/versioned"
+	dpInformer "github.com/kosmos.io/kosmos/pkg/generated/informers/externalversions"
```

Comment:
It‘s not necessary to use alias here?
```suggestion
"github.com/kosmos.io/kosmos/pkg/generated/informers/externalversions"
```